### PR TITLE
Consistent returns

### PR DESCRIPTION
**Pull request by bjpirt**
# What?
Use more consistent returns for the api use cases - finishing up the refactoring we were looking at last week

# Why?

# Checklist

- [ ] Unit tests
- [ ] Integration tests
- [ ] Cypress tests

# Notes

# Next steps

### CODE REVIEW

File: src/pages/api/tasks/[id]/launch.ts [hunk 1]
```diff
@@ -21,8 +22,8 @@ export default async (
   if (!tokenPayload) return res.status(500).end();
 
   const url = await getExternalProcessUrl.execute(id, tokenPayload.email);
-  if (url.error) return res.status(500).end();
+  if (isError(url)) return res.status(500).end();
 
-  res.writeHead(302, { Location: url.body });
+  res.writeHead(302, { Location: url }).end();
```
Comment (LBHTKarki) [inline src/pages/api/tasks/[id]/launch.ts:27]: this is likely to cause issues, but if we take what's in master at the moment that should resolve it. I noticed the header needs to be set separately before the result is returned with end()

File: src/usecases/api/setupUser.test.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+describe('SetupUser', () => {
+  it('should have tests', () => {});
```
Comment (LBHTKarki) [inline src/usecases/api/setupUser.test.ts:2]: that's for later I suppose

**Review by LBHTKarki — APPROVED** [review]
Looks good

File: src/pages/api/tasks/[id]/launch.ts (hunk 1 above)
Comment (bjpirt) [reply src/pages/api/tasks/[id]/launch.ts]: My bad - I thought I'd rebased but had done it wrong - good catch. Have updted now

File: src/usecases/api/setupUser.test.ts (hunk 2 above)
Comment (bjpirt) [reply src/usecases/api/setupUser.test.ts:2]: yes - I just put it here as a reminder

**Pull request merged**